jenkins/build_server_unix.sh

```diff
@@ -174,21 +177,25 @@ create_prop () {
 
 prep_artifacts () {
     # Prepare artifact directory for latestbuild publishing
-    pushd ${ARTIFACTS_DIR}
 
-    # Nexus strips "release" from ${PRODUCT}-${OS}-${VERSION}-release.${PKG_TYPE}.  It also transforms
-    # ${PRODUCT}-${OS}-${VERSION}-<debug|release>-symbols.${PKG_TYPE} to 
-    # ${PRODUCT}-${OS}-${VERSION}-symbols-<release|debug>.${PKG_TYPE}.  In order to minic this on
+    # Nexus strips "release" from ${PRODUCT}-${OS}-${SHA_VERSION}-release.${PKG_TYPE}.  It also transforms
+    # ${PRODUCT}-${OS}-${SHA_VERSION}-<debug|release>-symbols.${PKG_TYPE} to
+    # ${PRODUCT}-${OS}-${SHA_VERSION}-symbols-<release|debug>.${PKG_TYPE}.  In order to minic this on
     # latestbuild, these pkgs are renamed accordingly during copy
 
-    cp ${WORKSPACE}/${PRODUCT}-${OS}-${VERSION}-debug.${PKG_TYPE} ${PRODUCT}-${OS}-debug.${PKG_TYPE}
-    cp ${WORKSPACE}/${PRODUCT}-${OS}-${VERSION}-release.${PKG_TYPE} ${PRODUCT}-${OS}.${PKG_TYPE}
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-debug.${PKG_TYPE} ${ARTIFACTS_SHA_DIR}/${PRODUCT}-${OS}-debug.${PKG_TYPE}
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-release.${PKG_TYPE} ${ARTIFACTS_SHA_DIR}/${PRODUCT}-${OS}.${PKG_TYPE}
+
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-debug.${PKG_TYPE} ${ARTIFACTS_BUILD_DIR}/${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-debug.${PKG_TYPE}
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-release.${PKG_TYPE} ${ARTIFACTS_BUILD_DIR}/${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-release.${PKG_TYPE}
```

Comment:
This is not correct, see the previous version removes the "release" from the end.

Reply:
okay.  changed ${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-release.${PKG_TYPE} to ${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}.${PKG_TYPE}